scalafmt-tests/src/test/resources/scala3/OptionalBraces_keep.stat

```diff
@@ -3580,3 +3580,83 @@ object a {
   val a = test("" +: Seq("").map { _ + "" }: _*)
   val a = test(("" +: Seq("").map { _ + "" })*)
 }
+<<< coloneol in fewer braces 1
+maxColumn = 80
+===
+object a:
+  def f(): Unit =
+    List(1, 2, 3).foldLeft(1):
+      case (a, b) => a + b
+>>>
+object a:
+   def f(): Unit =
+     List(1, 2, 3).foldLeft(1):
+        case (a, b) => a + b
+<<< coloneol in fewer braces 2
+maxColumn = 80
+===
+object a:
+  def f(): Unit =
+    List(1, 2, 3).foo: a =>
+      a + 2
+    .apply:
+      12 + 3
+>>>
+object a:
+   def f(): Unit =
+     List(1, 2, 3).foo: a =>
+        a + 2
+       .apply:
+          12 + 3
```

Comment:
```suggestion
   def f(): Unit =
     List(1, 2, 3).foo: a =>
        a + 2
     .apply:
          12 + 3
```
?

Reply:
it comes from the rule which adds indent to first NL before a dot (so that it's indented relative to the beginning of the full expression), and your proposal will be inconsistent with the formatting so far:
```
   // short maxColumn
   def f(): Unit =
     List(1, 2, 3).foo(a =>
       a + 2)
       .apply(
         12 + 3)
   // longer maxColumn
   def f(): Unit =
     List(1, 2, 3).foo(a => a + 2)
       .apply(12 + 3)
   // there's no NL before a dot
   def f(): Unit =
     List(1, 2, 3).foo { a =>
       a + 2
     }.apply {
       12 + 3
     }
   // there's a NL before a dot
   def f(): Unit =
     List(1, 2, 3).foo { a =>
       a + 2
     } //
       .apply {
         12 + 3
       }
```
i didn't like this formatting for `fold` (no proper unindent before select) and forced NL before the first dot if there's a second select. i could do the same for `classic` (but not `keep`).

i could try formatting it like this if you prefer:
```
   // short maxColumn
   def f(): Unit =
     List(1, 2, 3).foo: a =>
         a + 2
       .apply:
         12 + 3)
   // but, should we still do this?
   def f(): Unit =
     List(1, 2, 3).foo: a =>
       a + 2
```

Reply:
Ach right, I didn't realize it.